calico-enterprise/operations/monitor/metrics/recommended-metrics.mdx

```diff
@@ -0,0 +1,574 @@
+| Explanation                      | CPU in use by calico-node represented as a percentage of a core. |
+| Threshold value recommendation   | A spike at startup is normal. It is recommended to first achieve a baseline and then monitor for any unexpected increases from this baseline. Investigage if maintained CPU usage goes above 90%. |
+| Threshold breach symptoms        | Unexpected maintained CPU usage could cause Felix to fall behind and could cause delays to policy updates. |
+| Threshold breach recommendations | Check CPU usage on Kubernetes nodes. Increase resources if needed, rollout restart calico-node(s) if needed. |
+| Priority level                   | Recommended.                                                 |
+
+### Memory usage
+
+| Memory usage                     |                                                              |
+| -------------------------------- | ------------------------------------------------------------ |
+| Metric                           | <code>process_resident_memory_bytes</code>                   |
+| Example value                    | <code>process_resident_memory_bytes{endpoint="metrics-port", **instance**="10.0.1.20:9091", **job**="felix-metrics-svc", **namespace**="calico-system", **pod**="calico-node-qzpkt", **service**="felix-metrics-svc"} 98996224</code> |
+| Explanation                      | Amount of memory in use by  calico-node.                     |
+| Threshold value recommendation   | Recommended to achieve a baseline first, then monitor for any unexpected increases from this baseline. Investigate if maintained CPU usage goes above 90% of what is available from the underlying node. |
+| Threshold breach symptoms        | Unexpected, maintained, memory usage could cause Felix to fall behind and could cause delays to policy updates. |
+| Threshold breach recommendations | Check memory usage on Kubernetes nodes. Increase resources if needed, rollout restart typha(s) if needed. |
+| Priority level                   | Recommended.                                                 |
+
+### Active hosts on each node
+
+| Active hosts on each node        |                                                              |
+| -------------------------------- | ------------------------------------------------------------ |
+| Metric                           | <code>felix_active_local_endpoints</code>                    |
+| Example value                    | <code>felix_active_local_endpoints{**endpoint**="metrics-port", **instance**="10.0.1.30:9091", **job**="felix-metrics-svc", **namespace**="calico-system", **pod**="calico-node-6pcqm", **service**="felix-metrics-svc"} 36</code> |
+| Explanation                      | Number of active **pod-networked** pods, and HEPs, on this node. |
```

Comment:
```suggestion
| Explanation                      | Number of active **pod-networked** pods, and host endpoints, on this node. |
```
Feel like we shouldn't abbreviate HEP in the docs, users might not know what it means?